PySDM/physics/saturation_vapour_pressure/bolton_1980.py

```diff
@@ -0,0 +1,28 @@
+"""
+[Bolton 1980](https://doi.org/10.1175/1520-0493(1980)108<1046:TCOEPT>2.0.CO;2)
+"""
+
+import numpy as np
+
+
+class Bolton1980:
+    def __init__(self, _):
+        pass
+
+    @staticmethod
+    def pvs_Celsius(const, T):
+        """valid for 0 <= T <= 100 C, eq (9)"""
+        return (
+            np.exp((T**-2) * const.B80W_G0)
```

Comment:
to make it work, the whole result must be multiplied by a constant that is equal to `1 * si.mb` - this will give it a proper dimension and take into account that the formula in the paper has a milibar unit

Reply:
done